w3c.json

```diff
@@ -0,0 +1,5 @@
+ {
+    "group":      109735
```

Comment:
```suggestion
    "group":      87846
```

Reply:
Actually, I supposed so at the first moment, but I'd want to hear from chairs.. (@AdaRoseCannon  or?)

Reply:
I have no idea to be honest, sorry.

Reply:
ah, sorry, I was not clear on this. This value is W3C's group ID, which mean 109735 for IWWG, 87846 for IWCG.
I thought this polyfill is for WG product by WG, but polyfill itself is not a spec nor product (spec, registry, or note) controlled by IP and could be owned by CG.